l1-contracts/contracts/bridge/L1SharedBridge.sol

```diff
@@ -212,16 +210,19 @@ contract L1SharedBridge is IL1SharedBridge, ReentrancyGuard, Ownable2StepUpgrade
     ) external payable virtual onlyBridgehubOrEra(_chainId) whenNotPaused {
         (address l1Asset, bytes32 assetInfo) = _getAssetProperties(_assetInfo);
         _transferAllowanceToNTV(assetInfo, _amount, _prevMsgSender);
-        IL1StandardAsset(l1Asset).bridgeBurn{value: msg.value}(
-            _chainId,
-            0,
-            assetInfo,
-            _prevMsgSender,
-            abi.encode(_amount, address(0))
-        );
+        /* solhint-disable no-unused-vars */
+        bytes memory bridgeMintData = IL1StandardAsset(l1Asset).bridgeBurn{value: msg.value}({
+            _chainId: _chainId,
+            _mintValue: 0,
+            _assetInfo: assetInfo,
+            _prevMsgSender: _prevMsgSender,
+            _data: abi.encode(_amount, address(0))
+        });
 
         // Note that we don't save the deposited amount, as this is for the base token, which gets sent to the refundRecipient if the tx fails
         emit BridgehubDepositBaseTokenInitiated(_chainId, _prevMsgSender, _assetInfo, _amount);
+        // We could use this data instead of the above
+        // emit BridgehubMintData(bridgeMintData);
```

Comment:
There's probably a better way to use this data

Reply:
I think it's more cohesive if we have events in sharedbridge, so we're indexing in 1 place

By the way, do we need to add l2Receiver in L1NTV?

I would actually not create any variable in memory to make the call more efficient. 
If we have everything we need, then just suppress the other linting error (unused return value)